ssm/ssm2.py

```diff
@@ -485,6 +485,7 @@ def send_all(self):
                 while self._last_msg is None:
                     if not self.connected:
                         raise Ssm2Exception('Lost connection.')
+                    time.sleep(0.1)
```

Comment:
Maybe a comment as to why the `sleep` is needed?

Reply:
I believe the point is that when you are running a `while` loop you don't want to hammer out the retries as that's a waste of CPU resources. I'll come up with a comment.

Reply:
Actually doing some testing, only a very small sleep is required to avoid hogging CPU, so I might reduce it.

Reply:
...but we may then need to keep the sleep below in for rate limiting purposes. Testing it against the brokers is probably the easiest way to see the difference.

Reply:
Comment added and sleep reduced.